### DISCUSSION TITLE
Pe+ 13 build not fixed yet

### DISCUSSION THREAD

**Discussion by sanjeevstunner** [ROM-builders/temporary#18267] [Q&A] [answered]
https://github.com/ROM-builders/temporary/discussions/18212
Here it says fixed

But
https://cirrus-ci.com/task/4705282668888064?logs=build#L26

Remove pixel.xml from .repo/local_manifests directory

**Comment by Apon77** [comment, marked as answer]
https://cirrus-ci.com/task/4705282668888064?logs=sync#L21

**Reply by sanjeevstunner** [reply]
Thanks for pointing out. Should've failed tho 😅 as it didn't find the branch

**Reply by sanjeevstunner** [reply]
Have a good day 🤎
